dockerfiles/Caffe2Dockerfile

```diff
@@ -0,0 +1,24 @@
+ARG CODE_VERSION
+FROM clipper/py-rpc:${CODE_VERSION}
+
+COPY clipper_admin/clipper_admin/python_container_conda_deps.txt /lib/
+
+RUN echo deb http://ftp.de.debian.org/debian jessie-backports main >> /etc/apt/sources.list \
+      && apt-get update --fix-missing \
+      && apt-get install -yqq -t jessie-backports openjdk-8-jdk \
+      && conda install -y --file /lib/python_container_conda_deps.txt \
+      && conda install pytorch torchvision -c pytorch \
+      && conda install -c anaconda cloudpickle=0.5.2
+
+RUN conda install -c ezyang onnx \
+	&& RUN conda install -c conda-forge protobuf==3.4.0 \
```

Comment:
Remove the `RUN` directives after `&&`. Also, align `&&`'s with the ones on lines 7-11